standards/technical-approach.md

```diff
@@ -1,12 +1,32 @@
+## Audience
+
+TADs are ultimately technical documents for a technical audience, not for Product or non-technical stakeholders. They do require approval from the Engineering Leadership Team (ELT), which includes Engineering Managers and the Director of Engineering as [described below](#review-process).
+
+Generally, the sections of the TAD that you'll be writing are for (1) your technical peers, i.e. other engineers who will be reviewing PRs and implementing the feature alonside you, and (2) the Engineering Manager who will be writing the Review Summary section at the end of the review process. The Director of Engineering will be reading the Review Summary section and skimming any other sections of the document that they find helpful.
+
+Since the Engineering Manager might not be intimately familiar with your technical stack or the problem at hand, it is helpful to provide some additional context in the "Problem Definition/Goal" and "Architectural Summary" sections of the template. For example, an iOS engineer might add a brief sentence or two of background information about iOS technologies if the Engineering Manager is not familiar with those technologies.
 
 ## Authors
 
-TADs should be written by the most senior engineer who is or will be working on a project/feature/fix. If it is unknown who will be doing the implementation work for a project an engineer with previous experience or subject matter knowledge should be assigned the task of writing a TAD.
+TADs should be assigned to engineers by Technical Leads or Technical Architects. Writing a TAD is a good opportunity to grow as an engineer, to have sanctioned time to research integrating different approaches into an NYPL codebase, and experiment with a new tool or technology.
+
+If it is unknown who will be doing the implementation work for a project, an engineer with previous experience or subject matter knowledge should be assigned the task of writing a TAD.
+
+TADs SHOULD have one author. In instances where a more collaborative approach is desired, consider splitting the TAD up into separate TADs each with their own author. See more in the [length](#length) section below.
+
+## Tone
+
+TADs are persuasive documents grounded in facts and best practices. Writing a TAD is communicating, “This is what I think we should do, and this is why.” Readers SHOULD come away from the TAD understanding why we need the solution that you recommended.
+
+## Length
+
+TADs SHOULD be as concise as possible while including all of the details that have a material impact on your design's acceptability. Generally, we recommend a TAD be no longer than 10-12 pages.
```

Comment:
I'll often include appendices with supplemental data, code samples, and other asides that aren't essential reading (but linked from the body of the TAD to support certain ideas). I sometimes prefer this to placing that stuff in outside, linked documents (although I also sometimes link to outside, supporting spreadsheets and the like). I'm hoping folks will discount appendices content when assessing page count?

Reply:
Good point. I added this clarification. Agree that addendums are helpful and they shouldn't count toward the 10-12 page limit.